### DISCUSSION TITLE
What's the usage scenario difference between ucp_am_callback_t and ucp_am_recv_callback_t

### DISCUSSION THREAD

**Discussion by XCloudAI** [openucx/ucx#6944] [General]
In ucx API document file, the section 6.6.3.4 introuce ucp_am_callback_t and 6.6.3.5 introduce ucp_am_recv_callback_t. What's the usage scenario difference between them? There's no example code.

**Comment by yosefe** [comment]
Plse see example for [ucp_am_callback](https://github.com/openucx/ucx/blob/master/test/apps/iodemo/ucx_wrapper.cc#L592) and [ucp_am_recv_callback_t](https://github.com/openucx/ucx/blob/master/test/apps/iodemo/ucx_wrapper.cc#L774)
The former is for receiving active message notifications, and latter for receiving the payload of long active messages.

**Reply by XCloudAI** [reply]
Good example. Got it.
